.github/dependabot.yml

```diff
@@ -14,6 +14,7 @@ updates:
     directories:
       - "/packages/c/sshnpd/tools/"
       - "/packages/dart/sshnoports/tools/"
+      - "/tests/e2e_all/dockerfiles/"
```

Comment:
This will ensure we get PRs to keep hashes up to date as we create new base images